packages/plugin-core/src/commands/Goto.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  DendronError,
+  DVault,
+  NoteUtils,
+  RespV3,
+  VaultUtils,
+} from "@dendronhq/common-all";
+import _ from "lodash";
+import { DENDRON_COMMANDS } from "../constants";
+import { IDendronExtension } from "../dendronExtensionInterface";
+import { ExtensionProvider } from "../ExtensionProvider";
+import { getLinkFromSelectionWithWorkspace } from "../utils/editor";
+import { BasicCommand } from "./base";
+import { GotoNoteCommand } from "./GotoNote";
+import { GoToNoteCommandOutput, TargetKind } from "./GoToNoteInterface";
+import { OpenLinkCommand } from "./OpenLink";
+
+type CommandOpts = {};
+
+type CommandInput = {};
+
+type CommandOutput = RespV3<GoToNoteCommandOutput>;
+
+const GOTO_KEY = "uri";
+
+export class GotoCommand extends BasicCommand<CommandOpts, CommandOutput> {
```

Comment:
Can you add a small class header to indicate this command's relationship to `GoToNoteCommand`?